src/Generator/ClassGenerator.php

```diff
@@ -140,7 +141,17 @@ public static function fromReflection(ClassReflection $classReflection)
             }
 
             if ($reflectionMethod->getDeclaringClass()->getName() == $className) {
-                $methods[] = MethodGenerator::fromReflection($reflectionMethod);
+                $method = MethodGenerator::fromReflection($reflectionMethod);
+
+                if (self::CONSTRUCTOR_NAME === $method->getName()) {
```

Comment:
Should probably better moved to `MethodGenerator::isConstructor()`

Reply:
Would be fine to keep it as-is for now, IMO. Just `strtolower()` needed

Reply:
Added